src/app.css

```diff
@@ -2,96 +2,93 @@
-	--color-gray-950: 249 250 251;
-	--color-gray-900: 243 244 246;
-	--color-gray-800: 229 231 235;
-	--color-gray-700: 209 213 219;
-	--color-gray-600: 156 163 175;
-	--color-gray-500: 107 114 128;
-	--color-gray-400: 75 85 99;
-	--color-gray-300: 55 65 81;
-	--color-gray-200: 31 41 55;
-	--color-gray-100: 17 24 39;
-	--color-gray-50: 3 7 18;
+		/* Tailwind gray 950 -> 950  */
+		--color-base-50: 249 250 251;
+		--color-base-100: 243 244 246;
+		--color-base-200: 229 231 235;
+		--color-base-300: 209 213 219;
+		--color-base-400: 156 163 175;
+		--color-base-500: 107 114 128;
+		--color-base-600: 75 85 99;
+		--color-base-700: 55 65 81;
+		--color-base-800: 31 41 55;
+		--color-base-900: 17 24 39;
+		--color-base-950: 3 7 18;
 
-	--color-neutral-950: 250 250 250;
-	--color-neutral-900: 245 245 245;
-	--color-neutral-800: 229 229 229;
-	--color-neutral-700: 212 212 212;
-	--color-neutral-600: 163 163 163;
-	--color-neutral-500: 115 115 115;
-	--color-neutral-400: 82 82 82;
-	--color-neutral-300: 64 64 64;
-	--color-neutral-200: 38 38 38;
-	--color-neutral-100: 23 23 23;
-	--color-neutral-50: 10 10 10;
+		/* Tailwind green 950 -> 50  */
+		--color-primary-50: 240 253 244;
+		--color-primary-100: 220 252 231;
+		--color-primary-200: 187 247 208;
+		--color-primary-300: 134 239 172;
+		--color-primary-400: 74 222 128;
+		--color-primary-500: 34 197 94;
+		--color-primary-600: 22 163 74;
+		--color-primary-700: 21 128 61;
+		--color-primary-800: 22 101 52;
+		--color-primary-900: 20 83 45;
+		--color-primary-950: 5 46 22;
+	}
 
-	--color-slate-950: 248 250 252;
-	--color-slate-900: 241 245 249;
-	--color-slate-800: 226 232 240;
-	--color-slate-700: 203 213 225;
-	--color-slate-600: 148 163 184;
-	--color-slate-500: 100 116 139;
-	--color-slate-400: 71 85 105;
-	--color-slate-300: 51 65 85;
-	--color-slate-200: 30 41 59;
-	--color-slate-100: 15 23 42;
-	--color-slate-50: 2 6 23;
-}
+	html.light {
+		color-scheme: light;
+		--background-gradient: theme('colors.neutral.100'), theme('colors.slate.100');
+		--gradient-primary: theme('colors.cyan.600'), theme('colors.green.600');
+
+		--color-white: 0 0 0;
+		--color-black: 255 255 255;
+
+		/* Tailwind gray 950 -> 50 */
+		--color-base-950: 249 250 251;
+		--color-base-900: 243 244 246;
+		--color-base-800: 229 231 235;
+		--color-base-700: 209 213 219;
+		--color-base-600: 156 163 175;
+		--color-base-500: 107 114 128;
+		--color-base-400: 75 85 99;
+		--color-base-300: 55 65 81;
+		--color-base-200: 31 41 55;
+		--color-base-100: 17 24 39;
+		--color-base-50: 3 7 18;
+
+		/* Tailwind green 950 -> 50  */
+		--color-primary-950: 240 253 244;
+		--color-primary-900: 220 252 231;
+		--color-primary-800: 187 247 208;
+		--color-primary-700: 134 239 172;
+		--color-primary-600: 74 222 128;
+		--color-primary-500: 34 197 94;
+		--color-primary-400: 22 163 74;
+		--color-primary-300: 21 128 61;
+		--color-primary-200: 22 101 52;
+		--color-primary-100: 20 83 45;
+		--color-primary-50: 5 46 22;
```

Comment:
I think it might be interesting to collapse these to have fewer options and making it easier to follow a "color system/design system", IDK if currently viable though